src/main/java/org/gridsuite/modification/server/modifications/GenerationDispatch.java

```diff
@@ -83,21 +83,35 @@ private static double computeTotalDemand(Component component, double lossCoeffic
 
     private static double computeTotalAmountFixedSupply(Network network, Component component, List<String> generatorsWithFixedSupply, Reporter reporter) {
         double totalAmountFixedSupply = 0.;
-
+        AtomicInteger numGeneratorsWithoutSetpoint = new AtomicInteger(0);
```

Comment:
Pas besoin d'utiliser un atomic. Lire la taille de generatorsWithoutSetpointList devrait suffire

Reply:
still have one commit not pushed, I thought that it was already done